From 1fb97d9ff639d2279e51c0fa30de5d3b05f2b0bb Mon Sep 17 00:00:00 2001
From: William Woodall <william@osrfoundation.org>
Date: Thu, 27 Apr 2017 11:52:32 -0700
Subject: [PATCH] add cmake rule to install package.xml

---
 CMakeLists.txt | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index bc562d0..fe871e4 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -137,7 +137,8 @@ ENDFOREACH(INC ${PC_INCLUDE_DIRS})
 CONFIGURE_FILE (${CMAKE_CURRENT_SOURCE_DIR}/stage.pc.in ${CMAKE_CURRENT_BINARY_DIR}/stage.pc @ONLY)
 INSTALL (FILES ${CMAKE_CURRENT_BINARY_DIR}/stage.pc DESTINATION ${PROJECT_LIB_DIR}/pkgconfig/)
 
-
+# Install catkin package.xml
+install(FILES package.xml DESTINATION share/stage)
 
 MESSAGE( STATUS "Installation path CMAKE_INSTALL_PREFIX=${CMAKE_INSTALL_PREFIX}" )
 
-- 
2.9.0

