From 54331d92fb309b5eeb53d8f4b2c8b9f9241df9cb Mon Sep 17 00:00:00 2001
From: William Woodall <wjwwood@gmail.com>
Date: Fri, 14 Mar 2014 13:02:12 -0700
Subject: [PATCH 04/11] quote prefix in stage.pc.in

---
 stage.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/stage.pc.in b/stage.pc.in
index 873fd27..8d63f01 100644
--- a/stage.pc.in
+++ b/stage.pc.in
@@ -1,4 +1,4 @@
-prefix=@CMAKE_INSTALL_PREFIX@
+prefix="@CMAKE_INSTALL_PREFIX@"
 
 Name: stage
 Description: Stage robot simulation program, C++ library and Player plugin - part of the Player Project (http://playerstage.org)
-- 
2.9.0

