From 406426db2642fd8dc4de2361b1cd8ae97ae7cbeb Mon Sep 17 00:00:00 2001
From: William Woodall <william@osrfoundation.org>
Date: Wed, 22 Mar 2017 12:02:06 -0700
Subject: [PATCH 11/11] fixing abs ambiguity compiler error

---
 libstage/world.cc | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/libstage/world.cc b/libstage/world.cc
index 351d843..af9eb49 100644
--- a/libstage/world.cc
+++ b/libstage/world.cc
@@ -83,6 +83,8 @@
 
 //#define DEBUG 
 
+#include <cstdlib>
+
 #include <stdlib.h>
 #include <assert.h>
 #include <string.h> // for strdup(3)
@@ -803,8 +805,8 @@ RaytraceResult World::Raytrace( const Ray& r )
   // Graphics Gems IV  
   const int32_t sx(sgn(dx));  
   const int32_t sy(sgn(dy));  
-  const int32_t ax(abs(dx)); 
-  const int32_t ay(abs(dy));  
+  const int32_t ax(std::abs(dx)); 
+  const int32_t ay(std::abs(dy));  
   const int32_t bx(2*ax);	
   const int32_t by(2*ay);	
   int32_t exy(ay-ax); // difference between x and y distances
-- 
2.9.0

